includes/admin/customers/customer-actions.php

```diff
@@ -231,10 +232,25 @@ function edd_add_customer_email( $args ) {
 				);
 
 			} else {
+				// Find out if this email is taken by a user or a customer and then include a link to the correct admin view.
+				$user_type = 'customer';
+				$user = new EDD_Customer( $email );
+
+				if( empty($user->email) ){
+
+					$user = get_user_by( 'email', $email );
+					$user_type = 'user';
+				}
+
+				$link = (($user_type === 'customer') ?
```

Comment:
The formatting on this seems off... `$link = ( (condition) ? case 1 : case 2 )`

This should really just be `$link = condition ? case 1 : case 2`